lte/gateway/python/magma/enodebd/dp_client.py

```diff
@@ -45,3 +53,81 @@ def get_cbsd_state(request: CBSDRequest) -> CBSDStateResult:
         )
         return CBSDStateResult(radio_enabled=False)
     return res
+
+
+def _indoorTobool(s: str):
+    return s.lower() in ['true', 't', '1', 'yes', 'indoor']
+
+
+def build_enodebd_update_cbsd_request(
+    serial_number: str,
+    latitude_deg: str,
+    longitude_deg: str,
+    indoor_deployment: str,
+    antenna_height: str,
+    antenna_height_type: str,
+    antenna_gain: str,
+    cbsd_category: str,
+) -> EnodebdUpdateCbsdRequest:
+    # cbsd category and antenna height type should be converted to lowercase
+    # for the gRPC call
+    antenna_height_type = antenna_height_type.lower()
+    cbsd_category = cbsd_category.lower()
+    # lat and long values are part of tr181 specification, but they are kept in device config
+    # transformed and eventually kept within the device config as strings representing degress
+    latitude_deg_float = float(latitude_deg)
+    longitude_deg_float = float(longitude_deg)
+
+    indoor_deployment_bool = _indoorTobool(indoor_deployment)
+    antenna_gain_float = float(antenna_gain)
+
+    # antenna height may be optional, and does not necessarily be available on the eNB
+    if antenna_height is not None:
```

Comment:
I'd rather do this to avoid duplication:
```
height_type = StringValue(value=antenna_height) if antenna_height else None
installation_param = InstallationParam(
    ....
    height_type=height_type,
)
```

Reply:
Done as suggested.